genesyscloud/responsemanagement_responseasset/resource_genesyscloud_responsemanagement_responseasset_utils.go

```diff
@@ -0,0 +1,48 @@
+package responsemanagement_responseasset
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"terraform-provider-genesyscloud/genesyscloud/provider"
+	"terraform-provider-genesyscloud/genesyscloud/util/files"
+)
+
+func responsemanagementResponseassetResolver(responseAssetId, exportDirectory, subDirectory string, configMap map[string]interface{}, meta interface{}) error {
+	sdkConfig := meta.(*provider.ProviderMeta).ClientConfig
+	proxy := getRespManagementRespAssetProxy(sdkConfig)
+
+	fileName := fmt.Sprintf("asset-%s.jpeg", responseAssetId)
```

Comment:
For users exporting assets, it looks like the filename is really useful for differentiating them, since they don't have a regular name. The only other way, from a terraform perspective, is to open the image files. If we could keep the filename in there, I think it would be better. So something like `"{the value of filename}-{responseAssetId}.jpeg"`
You'll just have to trim off any path directories that prefix the file name